File: C/Estrutura condicional/troco.c
// Biblioteca(s)
#include <stdio.h>

int main()
{
    // Variáveis
    double unidade, recebido, total, troco;
    int quantidade;

    // Entrada de dados
    printf("Qual e o preço unitario do produto? RS");
    scanf("%lf", &unidade);
    printf("Quantas quantidades? ");
    scanf("%d", &quantidade);
    printf("Quanto o(a) cliente pagou? ");
    scanf("%lf", &recebido);

    // Algoritmo matemático
    total = unidade * quantidade;
    troco = recebido - total;

    // Condicional
    if (troco > 0) {
        printf("O(a) cliente comprou no total RS%.2lf, pagou RS%.2lf e portanto o seu troco e de R$%.2lf.", total, recebido, troco);
    }
    else {
        troco = troco * -1; // Artifício que retira o sinal negativo
        printf("Dinheiro insuficiente. Faltam R$%.2lf.", troco);
    }

    return 0;
}

File: Java/Estrutura condicional/troco.java
// Importando biblioteca
import java.util.Scanner;

public class troco {

	public static void main(String[] args) {
		
		// Variável
		double unidade, recebido, total, troco;
		int quantidade;
		
		// Para facilitar a leitura
		Scanner sc = new Scanner(System.in);
		
		// Entrada de dados
		System.out.print("Qual é o preço unitário do produto? R$");
		unidade = sc.nextDouble();
		
		System.out.print("Quantas quantidades? ");
		quantidade = sc.nextInt();
		
		System.out.print("Quanto o(a) cliente pagou? ");
		recebido = sc.nextDouble();
		
		
		// Algoritmo matemático
		total = unidade * quantidade;
		troco = recebido - total;
		
		// Condicional
		if (troco > 0) {
			System.out.print("O(a) cliente comprou no total R$" + String.format("%.2f", total) + ", pagou R$" + String.format("%.2f", recebido) + " e portanto o seu troco é de R$" + String.format("%.2f", troco));
		}
		
		else {
			troco = troco * -1; // Artifício que retira o sinal negativo
			System.out.print("Dinheiro insuficiente. Faltam R$" + String.format("%.2f", troco));
		}
		
		sc.close(); // Boa prática de programação em Java
	}
}